projects/packages/my-jetpack/_inc/style.scss

```diff
@@ -0,0 +1,7 @@
+@import '@automattic/jetpack-base-styles/style';
```

Comment:
If we import this here, we should remove it from `MyJetpackScreen`, to avoid loading `base-styles` twice.